iommi/style_bootstrap5.py

```diff
@@ -0,0 +1,201 @@
+        ),
+        popper_js=Asset.js(
+            attrs=dict(
+                src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.10.2/dist/umd/popper.min.js",
+                integrity="sha384-7+zCNj/IqJ95wo16oMtfsKbZ9ccEh31eOz1HGyDuCQ6wgnyJNSYdrPa03rtR1zdB",
+                crossorigin="anonymous",
+            )
+        ),
+        js=Asset.js(
+            attrs=dict(
+                src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js",
+                integrity="sha384-QJHtvGhmr9XOIpI6YVutG+2QOK9T+ZnN4kzFN1RtK3zEFEIsxhlmWl5/YESvpZ13",
+                crossorigin="anonymous",
+            )
+        ),
+    ),
+    Container=dict(
+        tag='div',
+        attrs__class={
+            'container': True,
+            'mt-5': True,
+            'pt-5': True,
+        },
+    ),
+    Field=dict(
+        shortcuts=dict(
+            boolean=dict(
+                input__attrs__class={'form-check-input': True, 'form-control': False},
+                attrs__class={'form-check': True},
+                label__attrs__class={'form-label': True},
+                template='iommi/form/bootstrap/row_checkbox.html',
+            ),
+            radio=dict(
+                attrs__class={
+                    'mb-3': False,
+                    'form-check': True,
+                },
+                input__attrs__class={
+                    'form-check-input': True,
+                    'form-control': False,
+                },
+            ),
+        ),
+        attrs__class={
+            'mb-3': True,
+        },
+        input__attrs__class={
+            'form-control': True,
+            'is-invalid': lambda field, **_: bool(field.errors),
+        },
+        help__attrs__class={
+            'form-text': True,
+            'text-muted': True,
+        },
+        label__attrs__class={'form-label': True},
+        label__attrs__class__form_label=True, # need this to make class render
+        non_editable_input__attrs__class={'form-control': False},
+    ),
+    FieldGroup=dict(
+        tag='div',
+        attrs__class={'form-row': True},
+    ),
+    Action=dict(
+        shortcuts=dict(
+            # In bootstrap one must choose a button style (secondary, success, ...)
+            # otherwise the styling is roughly identical to text.
+            button__attrs__class={
+                'btn': True,
+                'btn-secondary': True,
+            },
+            primary__attrs__class={
+                'btn-primary': True,
+                'btn-secondary': False,
+            },
+            delete__attrs__class={
+                'btn-danger': True,
+                'btn-secondary': False,
+            },
+        ),
+    ),
+    Table=dict(
+        attrs__class__table=True,
+        attrs__class={'table-sm': True},
+    ),
+    Column=dict(
+        header__attrs__class={'text-nowrap': True},
+        shortcuts=dict(
+            select=dict(
+                header__attrs__title='Select all',
```

Comment:
Hmm. This title should be localized. I guess it's not in the bootstrap 4 style either?